assets/js/base/components/text-input/style.scss

```diff
@@ -75,3 +75,7 @@
 		margin-bottom: 0;
 	}
 }
+
+fieldset:disabled {
```

Comment:
Could this selector target `.wc-block-checkout-step` instead of the more broad `fieldset`? It could then be added to `<FormStep>`'s style. I think we should avoid tag selectors as much as possible, to prevent affecting styles not related to our blocks.